sktime/utils/validation/forecasting.py

```diff
@@ -328,13 +328,24 @@ def check_scoring(scoring):
     ----------
     TypeError
         if object is not callable from current scope.
+    NotImplementedError
+        if metric requires y_pred_benchmark to be passed
     """
     # Note symmetric=True is default arg for MeanAbsolutePercentageError
     from sktime.performance_metrics.forecasting import MeanAbsolutePercentageError
 
     if scoring is None:
         return MeanAbsolutePercentageError()
 
+    if scoring._tags["requires_y_pred_benchmark"]:
+        msg = "".join(
+            [
+                "Metrics requiring benchmark forecasts",
+                "are not fully implemented in the sktime API yet",
```

Comment:
Perhaps this error message?
"`scoring` requiring benchmark forecasts (`y_pred_benchmark`) are not fully supported yet. Please use a performance metric that does not require `y_pred_benchmark` as a keyword argument in its call signature."

Reply:
I've updated the error message.